tests/run_obs_testing.sh

```diff
@@ -16,7 +16,6 @@ DOCKERFILE_PATH="${SCRIPT_PATH}/Dockerfile"
 declare -a OS_DISTRIBUTION=( \
 	'ubuntu:16.04' \
 	'ubuntu:18.04' \
-	'ubuntu:19.04' \
```

Comment:
Fedora 29 is also EOL if you wanted to also remove that?

Reply:
@jodh-intel changes applied